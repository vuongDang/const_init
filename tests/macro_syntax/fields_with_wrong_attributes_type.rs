use const_init::ConstInit;

#[derive(ConstInit)]
struct FooBar {
    #[const_init(value = true)]
    foo: bool,
    #[const_init(value = true)]
    bar: usize,
}

fn main() {}
