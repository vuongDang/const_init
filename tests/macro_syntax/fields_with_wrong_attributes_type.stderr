error[E0308]: mismatched types
 --> tests/macro_syntax/fields_with_wrong_attributes_type.rs:7:26
  |
7 |     #[const_init(value = true)]
  |                          ^^^^ expected `usize`, found `bool`
